### PR TITLE
mrc-2408 Collapsible component for read-only parameters

### PR DESCRIPTION
**Pull request by EmmaLRussell**
Adds a `Collapsible` panel component to allow parameter forms along with their edit buttons to be collapsed together. We'll also use this to make Charts collapsible. Removes code making parameter groups collapsible using vue-dynamic-form's collapse functionality by doctoring the form metadata.

### CODE REVIEW

File: src/app/static/src/components/Collapsible.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+interface Props {
+    initialOpen: boolean,
+    heading: string
+}
+
+interface Data {
+    open: boolean
+}
+
+export default Vue.extend<Data, Methods, unknown, Props>({
```
Comment (mwoodbri) [inline src/app/static/src/components/Collapsible.vue:32]: Genuine question: it is necessary/useful to specify `Props` here even if we're already specifying the types inline below?

**Review by mwoodbri — APPROVED** [review]
Appears to work very well, besides a minor cosmetic glitch appearing in both Firefox and Edge: collapsing/expanding shifts the content down temporarily. Slightly distracting, so if this is easy to fix then that would be great, otherwise no worries at this point - I couldn't immediately see why it was happending or how to resolve.

**Review by LekanAnni — APPROVED** [review]

File: src/app/static/src/components/Collapsible.vue (hunk 1 above)
Comment (EmmaLRussell) [reply src/app/static/src/components/Collapsible.vue:32]: Typescript compilation fails when using `this.someProp` if the Props are not defined.

**Comment by EmmaLRussell** [comment]
> Appears to work very well, besides a minor cosmetic glitch appearing in both Firefox and Edge: collapsing/expanding shifts the content down temporarily. Slightly distracting, so if this is easy to fix then that would be great, otherwise no worries at this point - I couldn't immediately see why it was happending or how to resolve.

I think this may be because vue-dynamic form includes an empty  `<h3>` when form controlGroup does not have a label specified - will create a ticket.

**Pull request merged**